kernel/src/fs/configfs/tsm.rs

```diff
@@ -0,0 +1,206 @@
+// SPDX-License-Identifier: MPL-2.0
```

Comment:
Define a concept before using it. Throughout the PR, not a single of line cares to define what TSM is supposed to be.

```rust
// SPDX-License-Identifier: MPL-2.0

//! Trusted Security Manager (TSM).
//!
//! This module provides a cross-vendor ConfigFS interface
//! for generating confidential-computing attestation reports.
//! Userspace writes a request blob under `/sys/kernel/config/tsm/report/$name/inblob`
//! and reads back the signed report from `outblob`,
//! with optional knobs like `provider` and `generation`.
//!
//! For more information about the interface,
//! checkout Linux's [documentation](https://www.kernel.org/doc/Documentation/ABI/testing/configfs-tsm).
```